File: C++/Reverse Bits.cpp
class Solution {
  public:
    long long reversedBits(long long x)
 {
        long long ans = 0;
        for(int i=0;i<32;i++)
        {
            if(x & 1<<i)
                ans+= pow(2,31-i);
        }
        return ans;
  }
};

File: JAVA/Reverse Bits.java
class Solution {
    static Long reversedBits(Long x) {
      String s=Long.toBinaryString(x);
      StringBuffer sb=new StringBuffer(s);
      sb.reverse();
      for(int i=sb.length();i<32;i++){
          sb.append("0");
      }
      long number = Long.parseLong(sb.toString(), 2);
      return number;
    }
};
